### DISCUSSION TITLE
Internalization

### DISCUSSION THREAD

**Discussion by elinduxus** [drakkan/sftpgo#1568] [General]
hello good afternoon

I am implementing the Portuguese language, what would be the best procedure for implementation?

**Comment by drakkan** [comment]
Hello,

please take a look at the docs [here](https://github.com/drakkan/sftpgo/blob/main/docs/internationalization.md) before starting to translate SFTPGo. Thank you

**Reply by elinduxus** [reply]
Hi,
Thanks for answering.
We will collaborate on Crowdin to translate into Portuguese.
When would we have the translation available in the code?

**Comment by drakkan** [comment]
> Hi, Thanks for answering. We will collaborate on Crowdin to translate into Portuguese. When would we have the translation available in the code?

Please read the docs linked above. 
Internationalization is just an experiment, we want to understand how much the maintenance burden increases. We can't guarantee anything, sorry

**Reply by elinduxus** [reply]
Thanks!

**Reply by comradekingu** [reply]
@drakkan You want Weblate, not Crowdin. :)
I can help if you like.
